src/Makefile.am

```diff
@@ -74,7 +74,7 @@ libproj_la_SOURCES = \
 	jniproj.c pj_mutex.c pj_initcache.c pj_apply_vgridshift.c geodesic.c \
 	pj_strtod.c \
 	\
-	pj_observation.c
+	pj_observation.c pj_cart.c
```

Comment:
Yes, that was a bug. Actually, I think that 8 years ago, Gerald had planned to move all projection files into a proj_NAME.c namespace (cf. proj_etmerc.c and proj_rouss.c)

Doing that would also make it way easier to find the right file in a lexicographically ordered file open dialog.

But it would probably make it slightly harder to follow the development history of the code in git.